src/qt/bitcoingui.cpp

```diff
@@ -73,7 +74,8 @@ const std::string BitcoinGUI::DEFAULT_UIPLATFORM =
 BitcoinGUI::BitcoinGUI(interfaces::Node& node, const PlatformStyle *_platformStyle, const NetworkStyle *networkStyle, QWidget *parent) :
     QMainWindow(parent),
     m_node(node),
-    platformStyle(_platformStyle)
+    platformStyle(_platformStyle),
+    netStyle(networkStyle)
```

Comment:
Remove.

Reply:
It's needed...

Reply:
Sorry, forgot to delete this comment while reviewing..